templates/cache.conf.j2

```diff
@@ -0,0 +1,41 @@
+
+events {
+    worker_connections 1024;  # Defines the maximum number of simultaneous connections
+}
+
+http {
+    # Set up the cache directory and parameters
+    proxy_cache_path /var/cache/nginx levels=1:2 keys_zone=my_cache:10m inactive=48h use_temp_path=off;
+
+    # # Buffer settings for large file transfers
+    # client_max_body_size 12g;  # Allow for even larger file uploads (12GB here for safety margin)
+    # client_body_buffer_size 256k;  # Increased buffer for client request body
+    # proxy_buffer_size 128k;  # Buffer size for proxy connection header
+    # proxy_buffers 32 128k;  # Number of buffers and their size for proxy response
+    # proxy_busy_buffers_size 512k;  # Maximum buffer size for busy connections
+    
+    server {
+        listen 80;
```

Comment:
Should we change this to another port? 8089 or something else that's non-standard? This way we keep the flexibility to run another service that's external facing on 80/443 easily.